camel-sap/pom.xml

```diff
@@ -18,73 +24,367 @@
-		    version="[$(version;===;${fuse.osgi.version}),$(version;==+;${fuse.osgi.version}))"
-		</fuse.osgi.import.strict.version>
-		<fuse.osgi.import.default.version>[$(version;==;$(@)),$(version;+;$(@)))</fuse.osgi.import.default.version>
-		<fuse.osgi.import.defaults>
-		    org.springframework.osgi.*;version="[1.2,2)",
-		    org.springframework.*;version="[3.1,4)",
-		    org.apache.commons.logging.*;version="[1.1,2)",
-		    org.apache.camel.*;version="[2.13,3)",
-		    org.apache.felix.gogo.commands.*;version="[0.10,1)",
-		    org.apache.felix.service.command.*;version="[0.10,1)",
-		    org.osgi.service.component.*;version="[1.1,2.0)",
-		    scala*;version="[2.11,2.12)"
-		</fuse.osgi.import.defaults>
-		<fuse.osgi.import.before.defaults/>
-		<fuse.osgi.import.additional/>
-		<fuse.osgi.import.pkg>
-		    io.fabric8.*;${fuse.osgi.import.fabric.version},
-		    ${fuse.osgi.import.before.defaults},
-		    ${fuse.osgi.import.defaults},
-		    ${fuse.osgi.import.additional},
-		    *
-		</fuse.osgi.import.pkg>
-		<fuse.osgi.activator/>
-		<fuse.osgi.failok>false</fuse.osgi.failok>
-		<fuse.osgi.private.pkg></fuse.osgi.private.pkg>
-		<fuse.osgi.export>io.fabric8.*;version=${fuse.osgi.version};-noimport:=true</fuse.osgi.export>
-		<fuse.osgi.split.pkg>-split-package:=first</fuse.osgi.split.pkg>
-		<fuse.osgi.import>${fuse.osgi.import.pkg}</fuse.osgi.import>
-		<fuse.osgi.dynamic/>
-		<fuse.osgi.symbolic.name>${project.groupId}.${project.artifactId}</fuse.osgi.symbolic.name>
-		<fuse.osgi.exclude.dependencies>false</fuse.osgi.exclude.dependencies>
-		<fuse.osgi.resource>{maven-resources}</fuse.osgi.resource>
-		<fuse.osgi.embed.dependency>!*</fuse.osgi.embed.dependency>
-		<fuse.osgi.embed.transitive>false</fuse.osgi.embed.transitive>
-		<fuse.osgi.require.bundle />
-		<fuse.osgi.capabilities.provide />
-		<fuse.osgi.capabilities.require />
-			
+		<sap.idoc.lib.version>3.0.10</sap.idoc.lib.version>
+		<sap.jco.lib.version>3.0.11</sap.jco.lib.version>
+
+		<sap.redhat.version.postfix>redhat_2</sap.redhat.version.postfix>
+
+		<sap.idoc.wrapper.version>${sap.idoc.lib.version}.${sap.redhat.version.postfix}</sap.idoc.wrapper.version>
+		<sap.jco.wrapper.version>${sap.jco.lib.version}.${sap.redhat.version.postfix}</sap.jco.wrapper.version>
+
+		<jaxb.core.version>2.3.0.1</jaxb.core.version>
+		<mockito.version>1.9.5</mockito.version>
+		<powermock-version>1.5.6</powermock-version>
+		<hamcrest.version>1.3</hamcrest.version>
+		<hamcrest.integration.version>1.3</hamcrest.integration.version>
+
+		<eclipse.update.site.url>http://download.eclipse.org/releases/2019-03</eclipse.update.site.url>
+		<jbosstools-locus.site.url>http://repository.jboss.org/nexus/content/unzip/unzip/org/jboss/tools/locus/jbosstools-locus/1.9.0-SNAPSHOT/jbosstools-locus-1.9.0-SNAPSHOT-updatesite.zip-unzip/</jbosstools-locus.site.url>
+
+		<!-- upstream eclipse bundle versions -->
+		<emf.core.version>2.15.0</emf.core.version>
+		<emf.edit.version>2.14.0</emf.edit.version>
```

Comment:
from 2.8.0 to 2.14.0, seems a big increase for a release patch stream https://github.com/jboss-fuse/fuse-components/blob/94a5d126f71d7b9158210dd7c2248a345e4d428e/camel-sap/pom.xml#L126

I think it is used at runtime too, not only for SAP Tooling. If only SAP tooling, i think it is fine. If at runtime, I think it is not but worth checking with Productization team.